src/frontend-admin/src/components/App/App.module.css

```diff
@@ -16,3 +16,105 @@
 .appLink {
   color: #61dafb;
 }
+
+.navContainer {
+  background: white;
+  height: 140px;
+}
+
+nav {
+  width: 100%;
+  height: 68px;
+  background: #f4f4f4;
+
+  display: flex;
+  flex-direction: row;  
+  justify-content: flex-end;
+  z-index: 3;
+}
+
+nav h1 {
+  flex-grow: 1;
+  align-self: flex-start;
+  margin: auto;
+  margin-left: 100px;
+  
+  
+  color: #000000;	
+  /*font-family: CQL;	*/
+  font-size: 36px;	
+  font-weight: bold;	
+  letter-spacing: -1.38px;
+}
+
+.space {
+  height: 75px;
+}
+
+.circle {
+  width: 50px;
+  height: 50px;
+  background: #333;
+  border-radius: 50%;
+
+  position: absolute;
+  left: 355px;
+  z-index: 1;
+}
+
+.linkContainer {
+  align-self: flex-end;
+  margin-right: 60px;
+  background: #333;
+  border-radius: 0 18px 18px 18px; 
+  
+  display: flex;
+  align-content: center;
+
+  z-index: 2;
+  height: 80px;
+}
+
+.navTab {
+  width: 120px;
+  padding: 0px 35px 5px 35px;
+  color: #fff;
+  text-decoration:none;
+  font-size: 15px;
+  line-height: 18px;
+  background: #333;
+  border-radius: 0 18px 0px 18px;
+  display: flex;
+}
+
+#lastTab {
```

Comment:
Nice tip here: you can do `.navTab:last-child {}` instead of a special id placed on the last tab element.